woocommerce/woocommerce-gutenberg-products-block
# Patterns > Upload optimized version of placeholder image

## What

The placeholder image for the pattern `woocommerce-blocks/just-arrived-full-hero` wasn't optimized. This is an optimized version of it with a reduction of 94% in filesize. The dimensions of the image haven’t changed but its not 152kb rather than 2.7MB

Fixes #

## Why



The image was significantly large and could have performance implications.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create post or page
2. Add pattern `woocommerce-blocks/just-arrived-full-hero`
3. Check that image renders correctly still.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
